monitorcontrol/vcp/vcp_windows.py

```diff
@@ -121,6 +121,24 @@ def get_vcp_feature(self, code: int) -> Tuple[int, int]:
                 raise VCPError("failed to get VCP feature") from e
             return feature_current.value, feature_max.value
 
+        def get_vcp_capabilities(self):
+            """
+            thing
+            """
+            cap_length = DWORD()
+
+            # try:
+            ctypes.windll.dxva2.GetCapabilitiesStringLength(
+                HANDLE(self.handle), ctypes.byref(cap_length)
+            )
+            cap_string = (ctypes.c_char * cap_length.value)()
+            ctypes.windll.dxva2.CapabilitiesRequestAndCapabilitiesReply(
+                HANDLE(self.handle), cap_string, cap_length
+            )
+            # except ctypes.WinError as e:
+            #     raise VCPError("failed to get VCP feature") from e
```

Comment:
Will the above raise an error?
Remove or uncomment these lines.

Reply:
Commented it out durring debug and forgot. Thanks.